### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards> Expenses Report]: Name is not defined for the 'Expand and Collapsed' button present under Amount column.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9076]
### Target Platforms

UWP

### SDK Version

1.0.2.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Universal
OS Build: 24H2 (OS Build 26252.5000)
App Version: 1.0.2.0
Screen Reader: Narrator

**Repro Steps:**

1. Open the Adaptive Cards Universal application.
2. Adaptive Card UWP Visualizer screen starts appearing.
3. Navigate to Expenses Report.Json button by using Tab key and activate it.
4. Expenses Report adaptive card will appear on right sight of the pane.
5. Navigate to Expenses Report card by using Tab key.
6. Press the tab key and navigate to the 'Expand' button and observe the issue.

**Actual Result:**
While navigating the expenses report card, screen reader is not conveying the name of the 'Expand or collapse' button as no name is defined for the same. When the focus lands on it then it is announcing only as 'Button.'

Also, after activating the 'Expand or collapse' button screen reader does not convey the changed name information. It remains silent.

**Expected Result:**
Name should be defined for the Expand or collapse button so that when the screen reader focus lands on it then it should announce the information as 'Expand Air Travel expenses dollar 300 button.'

Also, when user activates Expand button then it should announce as 'Expanded' and vice versa.

**Note:**
Same issue is repro with 'Show History' button.
Same issue is repro with NVDA.

**User Impact:**
This will impact the screen reader user as they will not be able to know the name of the control due to this they will not able to get to know by activating the control what action will be performed.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/user-attachments/assets/290a88e4-6e3d-47d1-98f3-c69fdff1aa0d

![Image](https://github.com/user-attachments/assets/460db26f-6e58-416d-945a-53133f6d4022)

![Image](https://github.com/user-attachments/assets/24651ebe-8a9a-404a-bf9c-9e4433d8bb79)

### Card JSON

```JSON
NA
```

### Sample Code

Originally tracking with below bug - 

https://github.com/microsoft/AdaptiveCards/issues/8973

**Comment by vagpt** [comment]
This issue is not repro on the updated UI, hence closing this issue.

![Image](https://github.com/user-attachments/assets/0a9b1504-2913-46bc-9dbc-592827138742)

![Image](https://github.com/user-attachments/assets/de105a79-b3e7-435b-9660-b65b65b1e4e4)
